lubridate.Rmd

```diff
@@ -337,80 +335,98 @@ int_season_winter <- season_19_winter %--% (season_20_spring - minutes(1))
 Note that the `%--%` operator in the last line is similar to the calls of the `interval()` function in the lines above.
 
 ### Groupings
-What follows is preliminary step working with group_bys and renames in order to then apply the findings to seasons again. 
+What follows is an intermediate step working with `group_by()`s and `rename()`s. Subseqeuntly the findings towards seasons will be apllied again. 
 
-The following GROUP_BY command creates a unique grouping per hour and then calculates the average temperature for every hour, then pastes this value into each observation per hour.
+The following `group_by()` command creates a unique grouping per hour and then calculates the average temperature for every hour, then pastes this value into each observation per hour.
+Recall that `tag_int` ranges from `2020-03-01` to `2020-03-02`.
 ```{r create df_group of TAG for STUNDE/TAG - simple}
 df_sel <- df %>%
-  filter(MESS_DATUM >= int_start(tag_int) & MESS_DATUM <= int_end(tag_int))
+  filter(MESS_DATUM >= int_start(tag_int) & 
+           MESS_DATUM <= int_end(tag_int))
+
 df_group <- df_sel %>% 
```

Comment:
You write that you are calculating the average temperature for every hour in line 340, but there is no averaging happening here if I am not mistaken, thus `df_group` and `df_sel` are equal.